assets/css/blockchain.css.scss

```diff
@@ -1,31 +1,42 @@
 @import 'angular-csp';
-@import 'bootstrap-blockchain';
+//import global blockchain CSS
+@import '../../node_modules/blockchain-css/sass/bootstrap-blockchain';
+@import '../../node_modules/blockchain-css/sass/variables';
+@import '../../node_modules/blockchain-css/sass/fonts';
+
+//import global utilities
+@import '../../node_modules/blockchain-css/sass/utilities/general';
+@import '../../node_modules/blockchain-css/sass/utilities/colors';
+@import '../../node_modules/blockchain-css/sass/utilities/flex';
+@import '../../node_modules/blockchain-css/sass/utilities/spacing';
+@import '../../node_modules/blockchain-css/sass/utilities/borders';
+@import '../../node_modules/blockchain-css/sass/utilities/typography';
+
+//import global elements
+@import '../../node_modules/blockchain-css/sass/elements/buttons';
+@import '../../node_modules/blockchain-css/sass/elements/inputs';
+@import '../../node_modules/blockchain-css/sass/elements/forms';
+@import '../../node_modules/blockchain-css/sass/elements/links';
+
+//import global modules
+@import '../../node_modules/blockchain-css/sass/modules/header';
+@import '../../node_modules/blockchain-css/sass/modules/footer';
```

Comment:
Can you drop the node_modules part, so that it's just `@import blockchain-css/...`? If it doesn't work by default, pretty sure you can add something to the Gruntfile to include that dir when sass is looking for modules.